swimlane/core/cursor.py

```diff
@@ -28,15 +28,13 @@ class PaginatedCursor(Cursor):
     """Handle paginated lists, exposes hooks to simplify retrieval and parsing of paginated data"""
 
     default_limit = 0
-    page_size = 10
+    default_page_size = 10
```

Comment:
The point is not to have 10 as a size doing search. So it has to be modified either here or on application.records.search I believe (it's in the ticket). I don't see it increased anywhere. Reports should still have 10, the rest should have at least 1000.